src/components/AdminSettings.vue

```diff
@@ -12,16 +12,48 @@
 				</label>
 				<input id="collaboard-instance"
 					v-model="state.admin_instance_url"
+					:class="{ 'greyed-out-text': !usingCustomInstanceUrl, 'invalid-url': !isValidUrl(state.admin_instance_url) }"
 					type="text"
 					placeholder="https://..."
-					@input="onInput">
+					@input="onInput"
+					@blur="fillEmptyUrls">
+				<transition name="fade">
+					<div v-if="usingCustomInviteUrl && !usingCustomInstanceUrl" class="custom-address-notice">
+						<InformationOutlineIcon :size="20" class="icon" />
+						<span class="notice-text">
+							{{ t('integration_collaboard', 'You have specified a custom invite address. Did you forget to specify a custom instance address?') }}
+						</span>
+					</div>
+				</transition>
+			</div>
+			<div class="field">
+				<label for="collaboard-invite-url">
+					<EarthIcon :size="20" class="icon" />
+					{{ t('integration_collaboard', 'Default Collaboard invite url') }}
+				</label>
+				<input id="collaboard-instance"
+					v-model="state.admin_invite_url"
+					:class="{ 'greyed-out-text': !usingCustomInviteUrl, 'invalid-url': !isValidUrl(state.admin_invite_url) }"
```

Comment:
Same here, computed `isInviteUrlValid`.

Reply:
Fixed in https://github.com/nextcloud/integration_collaboard/pull/19/commits/0ced9cc9128977153b487efce91e544f246bc0d0